src/bridge/Makefile.am

```diff
@@ -116,6 +116,8 @@ libcockpit_bridge_a_SOURCES += \
 	$(NULL)
 endif
 
+if WITH_COCKPIT_BRIDGE
```

Comment:
Note that this still builds libcockpit-bridge.a (and all of the C files it contains). Not a biggie, but should also be rather easy to fix?

Reply:
I looked into this.  It's not easy.  In order to get this treated as a library we need to introduce it somehow, which we do via a `noinst_` target.  Those get built as part of `make all` even if they're not depended on by anything.

A quick look through the docs didn't turn up a way to get a `nobuild_noinst_` target...

We could do some more conditionals here to avoid building these, but it would be a little bit complicated: we want to build `libcockpit-bridge.a` even in the `--disable-bridge` case for `cockpit-pcp`.  If we had `--disable-bridge` AND `--disable-pcp` then we could probably exclude this, and also the related metrics library, but, honestly, I didn't think it was worth it for the added complexity.

Reply:
Agreed, this isn't worth wasting time on. I just wanted to make sure it's not a simple oversight.

Reply:
I actually took another look at this: it's `libcockpit-metrics.a` which is used by both `cockpit-pcp` and `cockpit-bridge` and would be difficult to conditionalize.  `libcockpit-bridge.a` is only used by the bridge and its tests and can very easily be tucked into the conditional.  Fixed that.